airflow/providers/sftp/hooks/sftp.py

```diff
@@ -320,3 +320,11 @@ def append_matching_path_callback(list_):
         )
 
         return files, dirs, unknowns
+
+    def test_connection(self) -> Tuple[bool, str]:
+        """Test the SFTP connection by checking if remote entity '/some/path' exists"""
+        try:
+            self.path_exists('/some/path')
```

Comment:
I think this can be made slightly cheaper to

```suggestion
            self.get_conn().pwd
```

Reply:
Thanks, that's a lot more elegant!